config/feature.json

```diff
@@ -1,5 +1,42 @@
 {
     "features": {
-        "FEATURE_1": false
+        "FEATURE_1": false,
+        "MKDOCS": {
```

Comment:
mkdocs is the tool, could we name the feature docs_integration ?  We will be having some discussions with business about the features and I am trying to find 'user friendly' terminology to help that conversation. If we can use the same terminology in the code it'll be helpful (and also means if we had to swap out mkdocs for a different doc builder, unlikely, but possible then we don't have to update the references).

Reply:
done